tests/foreman/ui/test_errata.py

```diff
@@ -880,7 +880,9 @@ def test_positive_apply_for_all_hosts(
         workflow='deploy-template',
         host_class=ContentHost,
         _count=num_hosts,
-        deploy_network_type='ipv6' if settings.server.is_ipv6 else 'ipv4',
+        # TODO(shwsingh): this is best effor for dualstack. This host deployment
```

Comment:
I think this should be the question for @SatelliteQE/team-phoenix

Reply:
ACK, agreed to move it to a fixture in a separate PR.
```suggestion
        # TODO(phoenix): this is best effor for dualstack. This host deployment
```

Reply:
Also a typo
```suggestion
        # TODO(shwsingh): this is best effort for dualstack. This host deployment
```

Reply:
Resolved in 62e7ab792